src/main/java/me/fredthedoggy/restpapi/SparkWrapper.java

```diff
@@ -12,11 +12,10 @@
 class SparkWrapper {
     Service http;
 
-    void create(int port, List<String> tokens) {
+    void create(int port, List < String > tokens) {
```

Comment:
Just nitpick but can you revert this 😁

Reply:
Done if you didn't get a notification with my latest commit.